src/routes/passage/[passageId]/+page.svelte

```diff
@@ -275,12 +283,40 @@
         }
     }
 
+    function showOrDismissBookChapterSelectorPane(show: boolean) {
+        if (show) {
+            bookChapterSelectorPane?.present({ animate: true });
+        } else {
+            bookChapterSelectorPane?.hide();
+        }
+    }
+
     function navbarTitle(
         resourceData: ResourceData | null,
         currentBible: FrontendBibleBook | undefined,
         selectedTab: PassagePageTab,
-        selectedStepNumber: number
+        selectedStepNumber: number,
+        bibleContents: ApiBibleContents | null
     ) {
+        if (bibleContents?.chapters?.length) {
```

Comment:
Could you move this to the `else` below, make it `else if`, and have the same logic without the extra layer of nesting?